CMakeLists.txt

```diff
@@ -260,7 +260,9 @@ if(PROJECT_SOURCE_DIR STREQUAL PROJECT_BINARY_DIR)
 endif()
 
 if (NOT CMAKE_BUILD_TYPE)
-    set( CMAKE_BUILD_TYPE Release)
+   set( CMAKE_BUILD_TYPE Release)
+elseif(CMAKE_BUILD_TYPE STREQUAL "PureDebug")
```

Comment:
Elemental used to use `{Pure,Hybrid}{Debug,Release}` to specify whether multithreading was expected and whether debug or release (and the ill-advised `EL_HYBRID` option is now separate). The point is that, if there are only four options, it makes sense to complain as early as possible if Elemental breaks if none of those four are used. (And we need to make sure that `MinSizeRel` and `RelWithDebInfo` both set `EL_RELEASE` to `ON`)